hydra-node/bench/tx-cost/Main.hs

```diff
@@ -20,7 +20,7 @@ import Options.Applicative (
   short,
   strOption,
  )
-import Plutus.Orphans ()
+import Hydra.Plutus.Orphans ()
```

Comment:
My rationale of naming these modules `Hydra.Plutus..` is to make it clear from which package they are coming from and not to have potential conflicts with further upstream packages. This also holds for naming the package `hydra-plutus-extras` instead of `plutus-extras`.

Reply:
I trust your judgment call 👍